terra_notebook_utils/cli/table.py

```diff
@@ -70,7 +71,7 @@ def fetch_drs_url(args: argparse.Namespace):
 
 @table_cli.command("put-row", arguments={
     "--table": dict(type=str, required=True, help="table name"),
-    "--id": dict(type=str, required=True, help="Row id. This should be unique for the table"),
+    "--row": dict(type=str, required=False, default=None, help="row name"),
```

Comment:
Do these still need to be unique?  If so, I thought that that was helpful info.  I might also document the behavior if this is not specified.

Reply:
Fair point. Will update.